interleave-in-parent.md

```diff
@@ -0,0 +1,172 @@
+---
+title: INTERLEAVE IN PARENT
+summary: Interleaving tables improves query performance by optimizing the key-value structure of closely related table's data.
+toc: false
+toc_not_nested: true
+---
+
+Interleaving tables improves query performance by optimizing the key-value structure of closely related tables, attempting to keep data on the same [key-value range](frequently-asked-questions.html#how-does-cockroachdb-scale) if it's likely to be read and written together.
+
+{{site.data.alerts.callout_info}}Interleaving tables does not affect their behavior within SQL.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## How do interleaved tables work?
+
+In the key-value store, interleaving tables writes data from one table (known as the **child**) directly into another (known as the **parent**) by matching the child table's Primary Key to the parent's.
+
+### Interleave Prefix
+
+For interleaved tables to have Primary Keys that can be matched, the child table must use the parent table's entire Primary Key as a prefix of its own Primary Key––these matching columns are referred to as the **interleave prefix**. It's easiest to think of these columns as representing the same data, which is usually implemented with Foreign Keys.
+
+{{site.data.alerts.callout_success}}To formally enforce the relationship between each table's interleave prefix columns, we recommend using <a href="constraints.html#foreign-keys">Foreign Key constraints</a>.{{site.data.alerts.end}}
+
+For example, if you want to interleave `orders` into `customers` and the Primary Key of customers is `id`, you need to create a column representing `customers.id` as the first column in the Primary Key of `orders`&mdash;e.g., with a column called `customer`. So the data representing `customers.id` is the interleave prefix, which exists in the `orders` table as the `customer` column.
+
+### Key-Value Structure
+
+When you write data into the child table, it is inserted into the key-value store immediately after the parent table's key matching the interleave prefix.
+
+For example, if you interleave `orders` into `customers`, the `orders` data is written directly within the `customers` table in the key-value store. The following is a crude, illustrative example of what the keys would look like in this structure:
+
+~~~
+/customers/1
+/customers/1/orders/1000
+/customers/1/orders/1002
+/customers/2
+/customers/2/orders/1001
+/customers/2/orders/1003
+...
+/customers/n/
+/customers/n/orders/<order belonging to n>
+~~~
+
+By writing data in this way, related data is more likely to remain on the same key-value range, which can make it much faster to read from and write to. Using the above example, all of customer 1's data is going to be written to the same range, including its representation in both the `customers` and `orders` tables.
+
+## When should I interleave tables?
```

Comment:
Similar stylistic feedback as above. I'd prefer something like **When to Interleave Tables**.